### PR TITLE
Why data storage is shifting to the blockchain

### LINKED ISSUE #6780: Why Data Storage is Shifting to the Blockchain
#### Proposed title of article
Why Data Storage is Shifting to the Blockchain
#### Proposed article introduction
When it comes to data storage solutions, there was a moment when traditional in-house data centers were phased out, and the cloud entered the scene as a viable option. However, things are progressing, and it may soon be the turn of distributed storage, which will be based on Blockchain or a Blockchain data storage solution.

The secure data storage market is currently dominated by huge third-party providers, and now it is one of the key targets for Blockchain developers. Soon, everyone could all have access to cheap, secure ways of handling even the most sensitive data, without needing to trust any third parties.
#### Key takeaways 

1. Blockchain storage vs. cloud storage
2. Costs implications of storing data on the blockchain
3. How blockchain is transforming cloud storage
4. Current blockchain storage products available

#### Article quality
The adoption of blockchain as a more secure, cost-friendly alternative data storage system is gaining momentum. It is, therefore, imperative to discuss this not so much explored topic to help get leaders prepared of what to expect of blockchain as a data storage solution.

### PR DESCRIPTION
**Pull request by Snzovi**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [ ] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #6780  

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by louisefindlay23** [comment]
Plagiarism check complete

File: content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Why Data Storage is shifting to the blockchain
+
+Some years ago, the most embraced form of data storage was the traditional in-house data centers. But it was not too long before cloud storage started to replace them. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```
Comment (mercymeave) [inline content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md:3]: The most embraced form of data storage was the traditional in-house data centers in the recent past. However, cloud storage started to replace it. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```suggestion
The most embraced form of data storage was the traditional in-house data centers in the recent past. However, cloud storage started to replace it. Cloud storage came with the promise of greater efficiency and more data storage capacity that attracted many businesses.
```

File: content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+In the same way, cloud storage is now facing competition from blockchain technology. Blockchain comes with the promise to solve data security challenges - an issue that cloud storage has found challenging to deal with exhaustively. With blockchain data storage, companies and individuals would enjoy the kind of data transparency not seen before. Thus, many companies are opting for this new technology with the hope that it could revolutionize data storage for the better.
```
Comment (mercymeave) [inline content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md:5]:
```suggestion
In the same way, cloud storage is now facing competition from blockchain technology. Blockchain promises to solve data security challenges - an issue that is a great threat to cloud storage. 

With blockchain data storage, companies and individuals will enjoy the desired data transparency. Thus, many companies are opting for this new technology with the hope of revolutionizing data storage.
```

File: content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+This article discusses blockchain as a more secure, cost-friendly alternative data storage system. It also gets leaders prepared of what to expect of blockchain as a data storage solution.
```
Comment (mercymeave) [inline content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md:7]:
```suggestion
This article discusses blockchain as a more secure, cost-friendly alternative data storage system. It also gets the reader to prepare for what to expect of blockchain as a data storage solution.
```

File: content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+In cloud storage, data is mutable, while in blockchain, data is immutable. The cloud storage system is set up to let you change the already stored data. These modifications affect how another user perceives the data after the changes. Users cannot adjust the already existing data in the blockchain because the system bars them. You cannot delete or modify entries already made, which helps secure data from tampering.
+
+Cloud storage relies on third parties to ensure data security. It provides services using the Software as a Service, Platform as a Service, and Infrastructure as a Service models. Blockchain chooses to guarantee data protection and safeguard its integrity without involving third parties.
+
+Besides, data in the cloud can exist as either public or private. However, in blockchain, data is always public. Blockchain embraces transparency, meaning that an entity cannot hide data from other users. It enables individuals on the blockchain to track how data moves from one point to another.
```
Comment (mercymeave) [inline content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md:20]: Hey, the best way to compare two technologies is using one characteristic and looking into it on both sides. For instance, if you take the mode of access, how does it occur in cloud storage and how different is it is from the blockchain.

Taking that approach makes your article smooth and add flow to your content.

File: content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+### Costs implications of storing data on the blockchain
+Unsurprising, such a storage technology that threatens to 'overthrow' cloud storage and offer a highly secure system has its share of financial challenges. In the last few years, the costs for implementing and running a blockchain system have [risen](https://content-blockchain.org/newsarchive/2017/07/20/exploding-costs-of-storing-data-on-a-blockchain/) tremendously. One reason for this is the increasing acceptance of blockchain technology has also led to a rise in data transmission and storage. Besides, the high demand for blockchain services has also driven up cryptocurrency prices.
```
Comment (mercymeave) [inline content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md:23]: This paragraph sounds a bit off. Consider re-writing and splitting into two to convert a single idea per paragraph.

File: content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+There are specific factors that keep the blockchain functioning normally. In the case of bitcoin, the nodes and miners are responsible. For Ethereum, Gas is the fuel that maintains its routine running. Miners receive payment for their services, and in the case of Ethereum, one must pay for Gas.
```
Comment (mercymeave) [inline content/articles/why-data-storage-is-shifting-to-the-blockchain/index.md:25]: How can you verify that the minors are responsible? 

`For Ethereum` ....... `and in the case of Ethereum`. Avoid unnecessary repetition.

How have the mentioned points contributed to the normal functioning of the blockchain?

**Review by mercymeave — CHANGES_REQUESTED** [review]
Great area to shed some light on @Snzovi 

However I noticed some mistakes from your article that needs to be addressed.

1. Avoid too many cliches. They make the article become shallow in meaning.
2.  Go over the article keenly to eliminate the common grammar mistakes that exist in the article. 
3.  I didn't go over the entire article once I NOTICED  THE MISTAKES WERE MANY.
> You might have used Grammarly but there are mistakes that are still visible. Correct them then we get to the review.

Thanks

**Comment by mercymeave** [comment]
Hello @Snzovi 

I will be closing this PR to allow you make the required revisions. We recommend our students to make revisions in their own forked repos to clear up the queue.

**Pull request closed without merging**